<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_QTopMenu</class>
 <widget class="QWidget" name="Test_QTopMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>825</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Escain::QTopMenu" name="Content" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Direction:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="Horizontal">
           <property name="text">
            <string>Horizontal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="Vertical">
           <property name="text">
            <string>Vertical</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QRadioButton" name="transversalCellNum1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="transversalCellNum3">
           <property name="text">
            <string>3</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="transversalCellNum2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QRadioButton" name="transversalCellNum4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Transversal cell number:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QRadioButton" name="cellSide15">
           <property name="text">
            <string>15x15</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="cellSide35">
           <property name="text">
            <string>35x35</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="cellSide25">
           <property name="text">
            <string>25x25</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Cell side size:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QRadioButton" name="tabMargin10">
           <property name="text">
            <string>10</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tabMargin0">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="tabMargin20">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tab margin:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tab min percentil:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabMinPercentil0">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabMinPercentil60">
           <property name="text">
            <string>60</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabMinPercentil100">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Show/hide general group:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="generalGroupShow">
           <property name="text">
            <string>Show</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="generalGroupHide">
           <property name="text">
            <string>Hide</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>General group additional item:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="generalGroupAdditionalItemAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="generalGroupAdditionalItemRemove">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Additional tab:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="additionalTabAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="additionalTabRemove">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tab additional group:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="additionalTabGroupAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="additionalTabGroupRemove">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Group additional item:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="additionalTabItemAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="additionalTabItemRemove">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_14" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Button label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonLabelEmpty">
           <property name="text">
            <string>&quot;&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonLabelShort">
           <property name="text">
            <string>Short</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonLabelLong">
           <property name="text">
            <string>Very long text label here</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_13" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tab label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabLabelEmpty">
           <property name="text">
            <string>&quot;&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabLabelShort">
           <property name="text">
            <string>File</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabLabelLong">
           <property name="text">
            <string>Very long text label here</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tab group label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabGroupLabelEmpty">
           <property name="text">
            <string>&quot;&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabGroupLabelShort">
           <property name="text">
            <string>Short</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tabGroupLabelLong">
           <property name="text">
            <string>Very long text label here</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QGridLayout" name="gridLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Enabled:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="enabledTrue">
           <property name="text">
            <string>true</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="enabledFalse">
           <property name="text">
            <string>false</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Escain::QTopMenu</class>
   <extends>QWidget</extends>
   <header>QTopMenu.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Horizontal</tabstop>
  <tabstop>Vertical</tabstop>
  <tabstop>transversalCellNum1</tabstop>
  <tabstop>transversalCellNum2</tabstop>
  <tabstop>transversalCellNum3</tabstop>
  <tabstop>transversalCellNum4</tabstop>
  <tabstop>cellSide15</tabstop>
  <tabstop>cellSide25</tabstop>
  <tabstop>cellSide35</tabstop>
  <tabstop>tabMargin0</tabstop>
  <tabstop>tabMargin10</tabstop>
  <tabstop>tabMargin20</tabstop>
  <tabstop>enabledTrue</tabstop>
  <tabstop>enabledFalse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
